content/en/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md

```diff
@@ -33,10 +33,10 @@ driver of the kubelet.
 
 {{< note >}}
 
-{{< feature-state for_k8s_version="v1.21" state="stable" >}}
+{{< feature-state for_k8s_version="v1.22" state="stable" >}}
```

Comment:
This seems odd. If the feature is already graduated, we should leave it marked as stable from v1.21.

Reply:
- 1.21: set to systemd for `kubeadm init` only
- 1.22: set to systems for `kubeadm`  (both `kubeadm init` and `kubeadm join`)

So your advice is to keep the feature stable version to 1.21?

Reply:
Yep. The meaning of this shortcode is: this feature has been _state_ since version _x_.

Reply:
If there's no relevant feature gate it might be more reasonable to remove the shortcode and use text to explain what versions provide what behavior.

Reply:
there is no feature gate for this, but the behavior between 1.22 and 1.21 changes.

> If there's no relevant feature gate it might be more reasonable to remove the shortcode and use text to explain what versions provide what behavior.

something like this can be done:

> In v1.22, if the user is not setting the `cgroupDriver` field under `KubeletConfiguration`, kubeadm will default it to `systemd`.

Reply:
As Tim says, without the feature gate, we may remove the feature state here.

Updated. Thanks.